ceph/ceph_cluster.yml

```diff
@@ -6,25 +6,29 @@ parameters:
  prefix: ceph
  numcpus: 2
  memory: 4096
- image: centos8
+ image: fedora32
  notify: false
  admin_password: password
  disks:
  - 150
- - 150
+ - 15
```

Comment:
It would be nice if we add a comment below disks as a hint on how to set the sahred folder param, somthing like this:
`# ceph_dev_folder: /path/to/your/local/ceph`.

Reply:
The explanation of this parameter is in the README file. I prefer do not use commented lines in the plan